cms/standard_pages/tests/test_models.py

```diff
@@ -0,0 +1,151 @@
+                            {featured_page_dict['value']['title']}
+                        </a>
+                        </h2>
+                    </div>
+                    <div class="ons-document-list__item-description">{featured_page_dict['value']['description']}</div>
+                </div>
+            </li>
+            </ul>
+        """
+        self.assertContains(response, featured_page_html, html=True)
+
+    def test_custom_featured_page_internal_page_is_displayed_correctly(self):
+        """Test that the custom featured items are displayed on the page."""
+        internal_page = InformationPageFactory(parent=self.page)
+
+        featured_page_dict = {
+            "type": "featured_page",
+            "value": {"page": internal_page.id},
+        }
+
+        self.page.featured_pages = [featured_page_dict]
+        self.page.save_revision().publish()
+
+        response = self.client.get(self.page_url)
+
+        self.assertEqual(response.status_code, 200)
+
+        featured_page_html = f"""
+            <ul class="ons-document-list">
+            <li class="ons-document-list__item ons-document-list__item--featured">
+                <div class="ons-document-list__item-content">
+                    <div class="ons-document-list__item-header">
+                        <h2 class="ons-document-list__item-title ons-u-fs-m ons-u-mt-no ons-u-mb-2xs">
+                        <a href="{internal_page.url}">
+                            {internal_page.title}
+                        </a>
+                        </h2>
+                    </div>
+                </div>
+            </li>
+            </ul>
+        """
+        self.assertContains(response, featured_page_html, html=True)
+
+    def test_get_formatted_related_links_list_works_for_internal_pages(self):
+        """Test that the links to internal pages rare returned
+        in a format that can be consumed by the Design System list component.
+        """
+        internal_page = InformationPageFactory(parent=self.page)
+
+        self.page.related_links = [{"type": "related_link", "value": {"page": internal_page.id}}]
+
+        formatted_related_links = self.page.get_formatted_related_links_list()
+
+        self.assertEqual(formatted_related_links, [{"title": internal_page.title, "url": internal_page.url}])
+
+    def test_get_formatted_related_links_list_works_for_external_pages(self):
+        """Test that the links to external pages are returned
+        in a format that can be consumed by the Design System list component.
+        """
+        self.page.related_links = [
+            {"type": "related_link", "value": {"title": "An external page", "external_url": "external-url.com"}}
+        ]
+
+        formatted_related_links = self.page.get_formatted_related_links_list()
+
+        self.assertEqual(formatted_related_links, [{"title": "An external page", "url": "external-url.com"}])
```

Comment:
Needs a test with `@override_settings(IS_EXTERNAL_ENV=True)` that checks we get a 200 for this page

```
    @override_settings(IS_EXTERNAL_ENV=True)
    def test_load_in_external_env(self):
        """Test the page loads in external env."""
        response = self.client.get(self.page_url)
        self.assertEqual(response.status_code, 200)
```

Reply:
[Improved tests](https://github.com/ONSdigital/dis-wagtail/pull/78/commits/ce09d7c32fb17fa2d359b682c30e5c92bbdf0ef1)